src/main/java/org/openrewrite/analysis/trait/expr/VarAccess.java

```diff
@@ -110,6 +111,50 @@ public J visitIdentifier(J.Identifier ident, List<VarAccess> varAccesses) {
         }.visit(scope.getValue(), varAccesses, scope.getParentOrThrow());
         return varAccesses;
     }
+
+    static Collection<Expr> findAllValues(Cursor scope, Variable variable) {
```

Comment:
I don't think this method should live here (on this interface) since it's not being used to find instances of `VarAccess` unlike the method above. Also, I would call it `findAssignedValues`.

Reply:
Maybe create a package-private class called `VariableUtil` and put this method there?